lingua_franca/parse.py

```diff
@@ -351,14 +225,31 @@ def get_gender(word, context="", lang=None):
              or None if unknown/or unused in the given language.
     """
 
-    lang_code = get_primary_lang_code(lang)
-
-    if lang_code in ["pt", "es"]:
-        # spanish follows same rules
-        return get_gender_pt(word, context)
-    elif lang_code == "it":
-        return get_gender_it(word, context)
-    # TODO: get_gender_xx for other languages
-    _log_unsupported_language(lang_code,
-                              ['pt', 'it', 'es'])
-    return None
+
+@localized_function()
+def is_fractional(input_str, short_scale=True, lang=None):
+    """
+    This function takes the given text and checks if it is a fraction.
```

Comment:
Happily done, though on reflection I'm not sure why this is public. The number extractors use it a lot, and it doesn't hurt to expose it, but I don't know who else would need this check.

Reply:
Yeah I figured it was public because "why not" rather than the expectation that people will use it. I'm sure there'll be some useful situation at some point in the future...